src/common/ddd/id.generator.ts

```
@@ -0,0 +1,7 @@
+import { ulid } from 'ulid';
+
+export class IdGenerator {
+  public static generate(): string {
+    return ulid();
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)